src/OmniSharp/MSBuild/SolutionFile/SolutionFile.cs

```diff
@@ -17,7 +17,7 @@ internal sealed partial class SolutionFile
         private readonly IEnumerable<ProjectBlock> projectBlocks;
         private readonly IEnumerable<SectionBlock> globalSectionBlocks;
 
-        public SolutionFile(
+        internal SolutionFile(
```

Comment:
I had a compilation error without that. Inconsisistent accessiblility because of SectionBlock and ProjectBlock.